compiler/passes/docs.cpp

```diff
@@ -295,6 +295,34 @@ std::string generateSphinxProject(std::string dirpath) {
   return std::string(moddir);
 }
 
+/* Checks for the valid Project Version (non-recursively). If an error occurs, exit and report
+ * error.
+ */
+char * checkProjectVersion(char * projectVersion) {
```

Comment:
This function is only used in this file, so it could be declared static to the file rather than in the .h file

Reply:
made the function static